projects/ngx-pagination-data-source/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ngx-pagination-data-source",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

You don't need to bump the version here manually, the project follows semantic versioning based on conventional commits. We'll squash this PR into the master branch and run `npm run bump` and it'll infer the new version based on the incorporated changes - which in this case would rather entail a new major version. Note that multiple PRs can go into a single release, so bumping the version individually in each one manually gets hard :)

```suggestion
  "version": "4.2.0",
```